OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -43,12 +43,14 @@
       <Private>False</Private>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Drawing" />
```

Comment:
For some reason it needs the reference in order to build. Otherwise I get these errors:
```
Error	CS0012	The type 'RectangleF' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.	OpenRA.Test	\OpenRA\OpenRA.Test\OpenRA.Game\SyncImplTest.cs	33
Error	CS0012	The type 'Rectangle' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.	OpenRA.Test	\OpenRA\OpenRA.Test\OpenRA.Game\SyncImplTest.cs	33
```